snap/snapcraft.yaml

```diff
@@ -0,0 +1,69 @@
+name: f3
+version: "1.0"
```

Comment:
Is this F3's version or the version of the snap package?

This Snap craft is going to build F3 from the `master` branch, isn't it? If so and the version field above is  F3's version, this field could be just "lastest". Another option is to ask Snap to get the version from the file `version.h`.

Reply:
Hello, this is a proposal for what the snap package version could look like. It is not a hard requirement, and you are welcome to make any changes as you see fit. Ideally, you will use this as a template, make changes as you see fit, and then, if you like, publish the snap to the Snap Store. In the Snap Store, developers create their own accounts, and publish their software directly.

Some links that you might want to look at (the store and the build system, but you can use any build system you like):

https://snapcraft.io/
https://snapcraft.io/build

Reply:
I am going to leave this pull request open with the hope someone will step up and take the responsibility to maintain F3 at the Snap Store. Although I cannot take on this responsibility, I'll support whoever decides to do so.

Reply:
I have raised this with the Snapcrafters team, to see whether we can adopt F3 under our umbrella.